### PR TITLE
Fixes add-on audio selector

### LINKED ISSUE #4644: Selecting/disabling Audio devices on add-ons doesn't work
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

I've noticed a couple of weird things happening around the audio devices on a add-on configuration panel.

- It seems to have auto-selected the first audio device, even if none is selected in the Hass.io Supervisor
- It doesn't change/read the actual audio device?
- I can't decouple/remove audio devices anymore. After a refresh, it is still be there. (which is not caused by the issues above).
- The save button looks/acts weird.

Screen recording:

## Expected behavior

Able to select/remove audio devices from an add-on.

## Steps to reproduce

Install an add-on with Audio Capabilities (e.g., Spotify Connect) and play with the audio devices.

## Environment

- Home Assistant release with the issue: 0.104.3 / Supervisor 196 (dev)
- Last working Home Assistant release (if known): Supervisor 195
- UI Type (States or Lovelace): n/a
- Browser and browser version: Version 79.0.3945.117 (Official Build) (64-bit)
- Operating system: macOS Mojave

## Problem-relevant configuration

```yaml
n/a
```

## Javascript errors shown in your browser console/inspector

```txt
Nothing.
```

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

- Use `iron-select` instead of `selected-item-changed` to trigger `_set*Device`

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml
For the most part yaml is not really needed here is it?
```

## Additional information

- This PR fixes or closes issue: fixes #4644
- This PR is related to issue: Nope
- Link to documentation pull request: Nope

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.

How? Why? Examples, please.
- [ ] Tests have been added to verify that the new code works.

***

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: hassio/src/addon-view/hassio-addon-audio.ts [hunk 1]
```diff
@@ -123,14 +123,14 @@ class HassioAddonAudio extends LitElement {
   }
 
   private _setInputDevice(ev): void {
-    const device = ev.detail.device;
+    const device = ev.detail.item.attributes.device.value;
```
Comment (bramkragten) [inline hassio/src/addon-view/hassio-addon-audio.ts]: set device as property and use `ev.detail.item.device` or use `ev.detail.item.getAttribute("device")`

Comment (ludeeus) [reply hassio/src/addon-view/hassio-addon-audio.ts]: Property looked cleaner.

Comment (bramkragten) [reply hassio/src/addon-view/hassio-addon-audio.ts]: Yeah but now your `attr-for-selected` is missing

Comment (ludeeus) [reply hassio/src/addon-view/hassio-addon-audio.ts]: Right...

**Review by balloob — APPROVED** [review]

**Pull request merged**